plugins/notifications-backend/src/service/router.ts

```diff
@@ -99,13 +100,22 @@ export async function createRouter(
     }
 
     const refs = Array.isArray(entityRef) ? entityRef : [entityRef];
+    const fields = [
+      'kind',
+      'metadata.name',
+      'metadata.namespace',
+      'relations',
+      'spec.owner',
```

Comment:
we do not want this one

Reply:
It's needed here https://github.com/backstage/backstage/pull/24120/files#diff-e2b0532e3202f4ab3b59bf9abd868e81fbe500bddf84148c467487447eb1b420R152

Reply:
That's still one more bad usage of it then. That needs to be removed too, I'd say. If there's a field like that, then some processor should have converted it into a relation which we can act upon. Relation-generating fields in the spec are useless to such a degree that we've debated whether we should actually forcibly delete them during processing so that they are literally impossible to read :)